packages/react/src/components/Code/Code.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, fireEvent } from '@testing-library/react';
```

Comment:
[`user-event`](https://testing-library.com/docs/user-event/intro/) should be used over `fireEvent` when possible.